app/commands/cart_services/resolve_cart_chassis.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+#
+# Copyright 2021 Victoria Garcia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+class CartServices::ResolveCartChassis
```

Comment:
I didn't see anything explaining what a `Chassis` was; hopefully that's covered below?

Reply:
So a CartChassis is a Plain Old Ruby Object(not stored in the database) that has room for two bins: One with status for_now and one with status for_later.  It does a bunch of intra-cart admin (separation of concerns, yay.) It's a key piece that allows cart_items to be saved for later without horribly complicating the payment process.